CMakeLists.txt

```diff
@@ -62,14 +62,6 @@ else()
   ecbuild_find_package( eckit VERSION 1.13.0 QUIET )
   ecbuild_find_package( oops QUIET )
 endif()
-# Optional: pygrib for python for GRIB1/GRIB2 input files
-if ( USE_PYGRIB )
```

Comment:
@srherbener tried to be thorough and did a:
```
find . -name CMakeLists.txt -exec grep -H "iodaconv_pygrib_ENABLED" {} +
```
down the test directory and there are no more use cases for the pygrib the only other converter using `.grib2` specifically is the MRMS one and it uses `eccodes` rather than the `pygrib` library:
https://github.com/JCSDA-internal/ioda-converters/blob/8ae8581f72d9a50cd9f589e1a858b0c562e2da52/src/mrms/mrms_grib2ioda.py#L17

please let me know if you think there are any pitfalls with this removal of the `USE_PYGRIB`

Reply:
I think it is safe to remove these converters and tests. My recollection is that they were temporary, and to be usable one had to install pygrib in their python environment (which was the purpose of having the USE_PYGRIB control). 

I think this is unlikely, but if something breaks after removing all of this, then we can address it then.